test/e2e/density.go

```diff
@@ -203,13 +199,21 @@ var _ = Describe("Density", func() {
 		// during the test so clean it up here
 		rc, err := c.ReplicationControllers(ns).Get(RCName)
 		if err == nil && rc.Spec.Replicas != 0 {
-			DeleteRC(c, ns, RCName)
+			By("Cleaning up the replication controller")
```

Comment:
Sorry, my diff collapsed some important lines, I didn't realize this WAS the clean up function. :)